presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,46 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Add a variant of :func:`strpos` to return the position of the N-th instance of the substring.
+* Add :func:`strrpos` that returns position of the N-th instance of substring from the back of a string.
+* Add aggregation function :func:`entropy`.
+* Add classification aggregation functions :func:`classification_miss_rate`, :func:`classification_precision`,
+  :func:`classification_recall`, :func:`classification_thresholds`.
+* Add maximum standard error to :func:`approx_set`.
+* Fix an issue where outer joins without equality predicates would sometimes run on a single node
+  when ``join_distribution_type`` was set to ``AUTOMATIC``.
+* Add encryption for spill files (see :doc:`/admin/spill`).
+* Fix equality inference for ``VARCHAR`` columns.  This improves performance for queries that have
+  both constant predicates and join predicates on a ``VARCHAR`` column.
+* Fix a null pointer exception during the stats collecting phase in query planning.
+* Introduce ``max_tasks_per_stage`` session property and ``stage.max-tasks-per-stage`` config property.
+  These properties limit the number of tasks per stage for grouped execution.
+
+Web UI Changes
+--------------
+* Add information about query warnings to the web UI.
+
+Raptor Changes
+--------------
+* Revert the change introduced in 0.219 to rebalance bucket assignment after restarting
+  the cluster. Automatic rebalancing can cause unexpected downtime when restarting the cluster
+  to resolve emergent issues.
+
+Hive Connector Changes
+----------------------
+* Decrease the amount of memory used by the coordinator when scheduling Hive splits.
+* Improve performance of writing large ORC files.
+
+SPI Changes
+-----------
+* Introduce ``commitPartition`` in ```Metadata`` and ``ConnectorMetadata``.
```

Comment:
Do we wanna elaborate a bit on `commitPartition` maybe? @shixuan-fan

Reply:
nit: there are three backquotes before `Metadata`. 

While trying to elaborate this, I think I forgot to add the SPI change related to `PageSinkProperties`. @rschlussel 

How about:
```
* Add ``PageSinkProperties`` for ``createPageSink`` in ``PageSinkProvider`` and ``ConnectorPageSinkProvider``. It contains ``partitionCommitRequired``, which is default to false for the existing connectors, and is further explained in ``commitPartition``. 
* Introduce ``commitPartition`` in ``Metadata`` and ``ConnectorMetadata``. This SPI is coupled with ``PageSinkProperties#partitionCommitRequired``, and is used by the engine to commit a partition of data to the target connector. The connector that implements this SPI should ensure that if ``PageSinkProperties#isPartitionCommitRequired`` is true in ``ConnectorPageSinkProvider#createPageSink``, the written data is not published until ``ConnectorMetadata#commitPartition`` is called. Also, it is expected for the connector to add ``SUPPORTS_PARTITION_COMMIT`` in ``Connector#getCapabilities``. 
```